pkg/scaffold/util/util.go

```diff
@@ -26,6 +26,7 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/resource"
 )
 
+// nolint:lll
 func GetResourceInfo(r *resource.Resource, repo, domain string, isMultiGroup bool) (resourcePackage, groupDomain string) {
```

Comment:
```suggestion
func GetResourceInfo(
	r *resource.Resource,
	repo,
	domain string,
	isMultiGroup bool,
) (resourcePackage, groupDomain string) {
```

Reply:
I one of the 100 comments was requested to keep it with no lint I will attend +/- or request.
Since your suggestion will no compile.

Reply:
I guess these 2 lines are causing the problem.
```
repo,
domain string,
```
I guess you can change it to:
```
repo string,
domain string,
```